src/createServer.js

```diff
@@ -1,3 +1,66 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+function createServer() {
+  const server = http.createServer((req, res) => {
+    const { url } = req;
+    const [text, queryString] = url.substring(1).split('?');
+    const params = new URLSearchParams(queryString);
+
+    const toCase = params.get('toCase');
+
+    const errors = [];
+
+    if (!text) {
+      errors.push({
+        message: 'Text to convert is required. Correct request is: '
+        + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    if (!toCase) {
+      errors.push({
+        message: '"toCase" query param is required. Correct request is: '
+        + '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+      });
+    }
+
+    const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+    if (!supportedCases.includes(toCase) && toCase) {
```

Comment:
Better move your data validation logic to a separate module, code will look cleaner :)